Android/app/src/androidTest/java/tie/hackathon/travelguide/Screens/MainPage.java

```diff
@@ -0,0 +1,29 @@
+package tie.hackathon.travelguide.Screens;
+
+import tie.hackathon.travelguide.R;
+import tie.hackathon.travelguide.Tests.Helpers;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static tie.hackathon.travelguide.Tests.Helpers.mediaItemPosition;
+
+/**
+ * Created by valentin.boca on 2/22/2018.
+ */
+
+public class MainPage {
+    public static void openNavDrawer() throws Exception {
+        Helpers.clickOnContentDesc(R.string.navigation_drawer_open);
+    }
+
+    public static void signOut() throws Exception {
+        Helpers.clickOnContentDesc(R.string.navigation_drawer_open);
+        Helpers.clickOnAView(R.id.nav_view, withId(R.id.design_navigation_view), 8);
+
+    }
+
+    public static void clickOn(int rid, int position) throws Exception {
```

Comment:
Numele metodei este foarte general. Pentru ce face, s-ar potrivi "clickOnItemWithIdAtPosition" 
O metoda de genul asta (dar care sa primeasca un Matcher<View> ca parametru) ar trebui in Helpers si de acolo o folosesti peste tot unde faci  ".perform(click());". Acelasi lucrcu si pentru ".perform(typeText(text));".